samples/sample_script.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env sh
+
+# set -xv
+
+# show usage and quit
+function usage()
+{
+  printf "usage: %s <command>\n" $(basename $0) >&2
+  printf "\n" >&2
+  printf "    <command> is one of: download, remove\n" >&2
+  printf "    <sample_name> is one of: iris, gsod, metriccentury\n" >&2
+  printf "\n" >&2
+  printf "    commands are:\n"
+  printf "        download [<version>] - downloads and mounts all sample data\n" >&2
```

Comment:
```suggestion
  printf "        download [<version>] - downloads and mounts all example data\n" >&2
```

I'm making this suggestion throughout because it matches the repo name.

Reply:
got them (all, I think?)

Reply:
should we rename the script itself, too?  `samples_script.sh` could become `examples_script.sh`.  users don't see it, but ...